index.bs

```diff
@@ -636,6 +639,7 @@ enum XRVisibilityState {
   attribute EventHandler onsqueezestart;
   attribute EventHandler onsqueezeend;
   attribute EventHandler onvisibilitychange;
+  attribute EventHandler ontargetframeratechange;
```

Comment:
From call: Whatever naming we choose for the `targetFrameRate` attribute above should also be reflected here, as this event indicates changes to the effective frame rate and not the developer-provided target frame rate. (Though the effective framerate may change in response to a change to the target framerate.)